packages/appeals-service-api/jest.config.js

```diff
@@ -9,12 +9,13 @@ module.exports = {
   coverageThreshold: {
     global: {
       branches: 92,
-      functions: 86,
-      lines: 86,
+      functions: 85,
+      lines: 85,
```

Comment:
these fell just under 86% as the number of lines in the project grew. No detrimental coverage impact here, afaik.

Reply:
Is there a unhappy path test you could add instead?

Reply:
How do you mean?

This is the difference we are talking about:

```
Statements   : 85.99% ( 540/628 )
Branches     : 92.9% ( 327/352 )
Functions    : 86.92% ( 113/130 )
Lines        : 85.94% ( 538/626 )
```

Reply:
Resolved, I had included my 4 line test file in the PR. That is now removed. This is no longer an issue. 

```
Statements   : 86.4% ( 540/625 )
Branches     : 92.9% ( 327/352 )
Functions    : 87.6% ( 113/129 )
Lines        : 86.36% ( 538/623 )
```

Reply:
Quite the opposite, coverage increased :)